cgroupstats.py

```diff
@@ -77,13 +79,22 @@ class CgroupstatsClient(object):
 
     def __init__(self, verbose=False):
         self.verbose = verbose
-        self.nlsock = netlink.NetlinkSocket()
-
-    def get_load_stats(self, fd):
-        data = CgroupstatsCmdAttrList(cgroupstats_cmd_attr_fd=fd)
-        message = CgroupstatsMessage(
-            'GET', flags=netlink.MessageFlags.REQUEST,
-            attr_list=data,
-        )
-        replies = self.nlsock.query(message)
-        return replies[0].get_attr_list().get('CGROUP_STATS')
+        self.nlsock = netlink.NetlinkSocket(verbose=verbose)
+
+    @contextmanager
+    def open_fd(self, path):
+        fd = os.open(path, os.O_RDONLY)
+        try:
+            yield fd
+        finally:
+            os.close(fd)
+
+    def get_load_stats(self, path):
+        with self.open_fd(path) as fd:
```

Comment:
@chantra I re-implemented the context manager because using just `open` didn't work for directories (and cgroup hierarchies are under directories). Here's an example of what I get:
```
Traceback (most recent call last):
  File "cgroupstats_dump.py", line 30, in <module>
    main(sys.argv)
  File "cgroupstats_dump.py", line 25, in main
    stats = c.get_cgroup_stats(args.path)
  File "/home/alex/gnlpy2/cgroupstats.py", line 90, in get_cgroup_stats
    with open(path, 'r') as f:
IOError: [Errno 21] Is a directory: '/sys/fs/cgroup/cpu'
```